core/templates/domain/skill/ConceptCardObjectFactory.ts

```diff
@@ -29,12 +29,12 @@ import { AppConstants } from 'app.constants';
 import { RecordedVoiceovers, RecordedVoiceoversObjectFactory,
   IRecordedVoiceOverBackendDict } from
   'domain/exploration/RecordedVoiceoversObjectFactory';
-import { SubtitledHtml, SubtitledHtmlObjectFactory,
-  ISubtitledHtmlBackendDict} from
-  'domain/exploration/SubtitledHtmlObjectFactory';
-import { WorkedExample, WorkedExampleObjectFactory,
-  IWorkedExampleBackendDict} from
-  'domain/skill/WorkedExampleObjectFactory';
+import {
+  SubtitledHtml, SubtitledHtmlObjectFactory, ISubtitledHtmlBackendDict
```

Comment:
Just wanted to check, can you add the "} from" in this line? I see that we usually split after "from" in other files.

Reply:
Done